blocks/editor-settings/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { createContext } from '@wordpress/element';
+
+/**
+ * The default editor settings
+ * You can override any default settings when calling initializeEditor
```

Comment:
This DocBlock feels outdated, particularly in its reference to `initializeEditor` of another module, and that we document only one of the many settings available (`alignWide`).